src/Hamiltonians/HubbardRealSpace.jl

```diff
@@ -141,15 +202,16 @@ end
 
 # struct ================================================================================ #
 """
-    HubbardRealSpace(address; geometry=PeriodicBoundaries(M,), t=ones(C), u=ones(C, C), v=zeros(C, D))
+    HubbardRealSpace(address; geometry=PeriodicBoundaries(M,), t=ones(C, D), u=ones(C, C), Δ=zeros(C, C), v=zeros(C, D))
```

Comment:
```suggestion
    HubbardRealSpace(address; geometry=PeriodicBoundaries(M,), t=ones(C, D), u=ones(C, C), v=zeros(C, D), vprime=zeros(C, C))
```

I'm not in favour of naming the interaction parameter \Delta for two reasons: For one it is better to avoid unicode names for arguments in library functions. The other reason is that \Delta is already used for a reduced interaction in the (mapped) spin chain models (and if I understand correctly the value is different). My suggestion `vprime` at least has a precedent that it was used in the literature before (for the t-V-V' model).